docker/Dockerfile.nvidia

```diff
@@ -16,41 +16,52 @@
 # 
 #   docker build --network=host --build-arg MPI_VER=4 --file docker/Dockerfile.nvidia --tag devito:nvidia .
 #
+# SQUASH:
+#   If you want a smaller image or to collapse/combine the smaller layers, try squashing your docker image
+#
+#   pip install docker-squash
+#   docker tag devito:nvidia devito:nvidia-pre
+#   docker-squash  -t devito:nvidia-pre devito:nvidia
+#
 # RUN: 
 #   docker run --gpus all --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 devito:nvidia
-#   docker run --gpus all --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 devito:nvidia python examples/seismic/acoustic/acoustic_example.py
+#   docker run --gpus all --rm -it devito:nvidia python examples/seismic/acoustic/acoustic_example.py
 #
 #   docker run --gpus all --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 --device=/dev/infiniband/uverbs0 --device=/dev/infiniband/rdma_cm  devito:nvidia 
 #
 # to run in user context on a cluster with shared filesystem, you can add the correct user config as docker options e.g.:
 #   docker run --gpus all --rm -it -v `pwd`:`pwd` -w `pwd` -u $(id -u):$(id -g) devito:nvidia python examples/seismic/acoustic/acoustic_example.py
 #
 ##############################################################
-FROM python:3.8
+#FROM python:3.9 
+FROM python:3.9-slim-bullseye 
  
 ENV DEBIAN_FRONTEND noninteractive 
   
 # nodesource: nvdashboard requires nodejs>=10  
 RUN echo 'deb [trusted=yes] https://developer.download.nvidia.com/hpc-sdk/ubuntu/amd64 /' > /etc/apt/sources.list.d/nvhpc.list && \
     apt-get update -y && \
     apt-get install -y -q \
-        apt-utils curl\   
-        vim && \
+        apt-utils curl wget \   
     curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
     apt-get install -y -q \
-        nodejs \
+#        nvhpc-22-3-cuda-multi \
```

Comment:
same I guess

Reply:
Two previous cuda version and everythng that goes with it. Takes a lot of space usually yeah and not needed.